pkg/helm/client_test.go

```diff
@@ -149,7 +150,14 @@ func Test_cleanUpGenericMap(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			req := require.New(t)
-			req.Equal(tt.want, cleanUpGenericMap(tt.in))
+			out, err := cleanUpGenericMap(tt.in)
+			req.NoError(err, "cleanUpGenericMap failed")
+			req.Equal(tt.want, out)
+
+			// ultimately helm calls k8syaml.Marshal so we must make sure that the output is compatible
```

Comment:
did you verify (even if manually) that this would've caught the issue fixed by https://github.com/replicatedhq/embedded-cluster/pull/1834?

Reply:
How can I verify? Can you provide reproduction steps? They are not in the original fix.